addons/mail/static/src/messaging/component/partner_mention_suggestion/partner_mention_suggestion.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="mail.messaging.component.PartnerMentionSuggestion" owl="1">
+        <t t-if="partner and composer">
+            <t t-if="partner.divider">
+                <div class="dropdown-divider" role="separator"/>
+            </t>
+            <a class="o_PartnerMentionSuggestion dropdown-item" t-att-class="{ 'active': partner.id === composer.active_id }" href="#" role="menuitem" t-att-data-id="partner.id" t-on-click="_onClickMentionItem" t-on-mouseover="_onMouseOverMention">
+                <PartnerImStatusIcon
+                    partnerMentionSuggestionDisplay="true"
+                    partnerLocalId="partner ? partner.localId : undefined"
+                />
+                <span class="o_PartnerMentionSuggestionName"><t t-esc="partner.name"/></span>
+                <t t-if="partner.email">
+                    <span class="o_PartnerMentionSuggestionEmail">(<t t-esc="partner.email"/>)</span>
+                </t>
```

Comment:
This looks like a "phrase" crafted with different parts, that will be very hard to translate and might fail to work with right-to-left. Do all languages apply ( ) like this?
To double check.